delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/ElasticSearchViews.scala

```diff
@@ -201,6 +201,7 @@ final class ElasticSearchViews private (
   )(implicit caller: Caller): IO[ViewResource] = {
     for {
       (iri, pc) <- expandWithContext(fetchContext.onModify, project, id)
+      _         <- validateNotDefaultView(iri)
```

Comment:
Same, it would be better to have it in the evaluate function

Reply:
Yeah, looking at the other checks, they are all related to the state. Personally, I think it's clearer to put the check in the main method

Reply:
Unfortunately this means I now need to make sure in the ElasticSearchViewsSpec that we create the default view (currently we don't do this)

Reply:
This ended up causing design issues around our project deletion code. We're going to merge what we have now, and when you're back I can show you what we came up against. I still have the code so it'll be easy to pursue if we can figure out a way forward